### PR TITLE
shared-images cleanup and docs update

### LINKED ISSUE #289: get rid of shared-images and related shared-service namespace
**Is your feature request related to a problem? Please describe.**
With the cleanup of the shared-images folder (related to pending task [#60](https://github.com/opendevstack/ods-quickstarters/issues/60) and already done task [#215](https://github.com/opendevstack/ods-core/issues/215)) we can now get rid of the `shared-images` and realted `shared-services` namespace

**Describe the solution you'd like**
Once [#60](https://github.com/opendevstack/ods-quickstarters/issues/60) is finished we can clean up shared-images, its realted documentation and also assure shared-services namespace is no longer existing/required.

**Describe alternatives you've considered**
No alternative

### PR DESCRIPTION
**Pull request by gerardcl**
closes #289

### CODE REVIEW

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -1132,6 +1132,8 @@ You can login in with the Crowd admin user you set up earlier.
 
 === Add shared images
 
+NOTE: This section is not required anymore from OpenDevStack 2.0
```
Comment (michaelsauter) [inline docs/modules/ROOT/pages/getting-started.adoc]: Antora is versioned, please kill the whole section.

Comment (henrjk) [reply docs/modules/ROOT/pages/getting-started.adoc]: There was another place where uploading templates is mentioned where it says it only applies before 1.2.x or similar. I could dig out the link for that.  So perhaps the docs needs some review to remove version references?

Comment (michaelsauter) [reply docs/modules/ROOT/pages/getting-started.adoc]: Would be nice to clean that up, yes.

**Comment by michaelsauter** [comment]
@gerardcl I merged this now, as it improves the situation a lot. Would still be nice to clean the docs up even further ... but that is a nice to have :)

**Pull request merged**